README.md

```diff
@@ -377,6 +380,24 @@ HLTV.getEvents().then(res => {
 
 ---
 
+### getOngoingEvents
+
+Parses the info from the `hltv.org/events` page
+
+| Option |                                       Type                                        | Default value |                                    Description                                    |
+| :----: | :-------------------------------------------------------------------------------: | :-----------: | :-------------------------------------------------------------------------------: |
+|  all  | boolean |       false       | Its an option to show all ongoing events |
```

Comment:
Instead of having an `all` option I think it would be better if the method returned all events and each event had a `featured` property that indicates whether it's part of the "Featured" tab.

Reply:
I edited the function and added the property ```today```. In my option, only today and all is important.